src/poetry/console/commands/version.py

```diff
@@ -80,27 +80,27 @@ def increment_version(self, version: str, rule: str) -> "Version":
         from poetry.core.semver.version import Version
 
         try:
-            version = Version.parse(version)
+            version_ = Version.parse(version)
```

Comment:
Same as above, I'd rather call this `parsed` or similar.

Reply:
almost did that originally